### ISSUE TITLE
UI/UX Enhancement: Go button

### ISSUE THREAD

**Issue by Gitoffthelawn** [Quicksaver/OmniSidebar#131]
The ability to include the "Go" button is nice.  What would be even better would be to add that as an option in the dropdown menu within the sidebar.

This would simplify and cleanup the sidebar's toolbar while still giving the user the useful functionality.

**Comment by Quicksaver** [comment]
Agreed, I've thought about this before, but never got around to it. I might not do it any time too soon either, as I'm sure you'll understand, I have to give this less priority than some other things. :)

**Comment by Gitoffthelawn** [comment]
Sounds good!  Thanks!
